src/templates/controller/controller.template.ts

```diff
@@ -66,25 +71,41 @@ class ControllerFileFactory {
         }
         if (pathParams) {
             this._imports['@nestjs/common'].add('Param')
+
+            if (pathParams.some((param) => param.schema.format == 'uuid')) {
+                this._imports['crypto'].add('UUID')
+            }
         }
 
         if (queryParams) {
             this._imports['@nestjs/common'].add('Query')
+
+            if (queryParams.some((param) => param.schema.format == 'uuid')) {
+                this._imports['crypto'].add('UUID')
+            }
         }
 
         const baseMethodName = methodNames[method.toUpperCase() as keyof typeof methodNames]
-        const byParamSuffix = pathParams ? `By${pathParams.map(_.capitalize).join()}` : ''
+        const byParamSuffix = pathParams ? `By${pathParams.map((pp) => _.capitalize(pp.name)).join()}` : ''
```

Comment:
Note: можно еще так реализовать:
```suggestion
        const byParamSuffix = pathParams ? `By${pathParams.map(({name}) => _.capitalize(name)).join()}` : ''
```